Twittimation/Commands/Exit.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+
+namespace Twittimation.Commands
+{
+    public class Exit : Command
```

Comment:
This command doesn't seem to make sense. Can it be deleted?

Reply:
Currently you can start the program with no commands, and then you can run any number of commands without it exiting, finally you can exit with this command.

Reply:
Program should be non-interactive.